keps/sig-api-machinery/3157-watch-list/README.md

```diff
@@ -887,6 +921,16 @@ logs or events for this purpose.
 
 ###### How can someone using this feature know that it is working for their instance?
 
+Assuming that historical data is available then comparing the number of LIST and WATCH requests to the server will tell whether the feature was enabled. 
+When this feature is enabled, the number of LIST requests will be smaller. 
+The difference primarily arises from switching informers to a new mode of operation.
+
+Checking whether `ENABLE_CLIENT_GO_WATCH_LIST_ALPHA` environment variable has
```

Comment:
We were using it for alpha, but we should get rid of it and depend on FG.

Reply:
Do we have to remove it ? I'd like to keep it. Developers could use this env var to enable the feature for their components.

Reply:
I'm confused.  Is this feature moving to beta in 1.29 or moving to alpha in 1.29?

Reply:
We should change the name. I will open a new PR for that.

Reply:
> Developers could use this env var to enable the feature for their components.

Reflector has a `UseWatchList` field to control it.
Enablement of this feature can be controller via component-level feature-gate [not client-go feature-gate] - that sounds desired to me.
I just don't like env-var - it's too easy to mess up potentially.

Reply:
requires update to doc with client featuregate to pass PRR